HomeWorks/HW04.Task5/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace HW04.Task5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            for (int i = Convert.ToInt32('Z'); i >= Convert.ToInt32('A'); i--)
```

Comment:
correct. but you do not need to convert it. you can use char instead of int.
just try it